### PR TITLE
Fcrepo 3912

### PR DESCRIPTION
**Pull request by Surfrdan**
Jira https://fedora-repository.atlassian.net/browse/FCREPO-3912

### CODE REVIEW

File: build-and-push-to-dockerhub.sh [hunk 1]
```diff
@@ -21,14 +31,8 @@ platforms=linux/arm64,linux/amd64
 # build and push images
 for docker_tag in "${@:2}"
 do
+    # Check that valid tag is supplied. If not, fail
```
Comment (tomcbe) [inline build-and-push-to-dockerhub.sh]: This comment is no longer valid here as the check was moved to lines 9-17.

**Review by tomcbe — CHANGES_REQUESTED** [review]
Looks good to me, except that the comment on line 35 is no longer valid

File: build-and-push-to-dockerhub.sh (hunk 1 above)
Comment (Surfrdan) [reply build-and-push-to-dockerhub.sh]: good spot

**Review by whikloj — APPROVED** [review]
👍

**Pull request merged**